### DISCUSSION TITLE
I want to switch to debug mode, but it gives me this error

### DISCUSSION THREAD

**Discussion by xeemoff** [modular/modular#2484] [Q&A]
error: JIT(0x7fffa7407000)(0x00007fffa7407000) The parsed MLIR structure type '!kgen.variant<struct<()>, scalar<bool>, scalar<si64>, scalar<f64>, struct<(pointer<none>, index)>, pointer<none>, pointer<none>, pointer<none>>' has not byte size. The MLIR type won't be used and regular MLIR-agnostic DWARF parsing will be performed.
error: JIT(0x7fffa7407000)(0x00007fffa7407000) [MojoDWARFParser::ParseTypeFromDWARF]: Unhandled type tag. Die = 0x403, tag = DW_TAG_variant_part, name = DW_TAG_variant_part
error: JIT(0x7fffa7407000)(0x00007fffa7407000) [MojoDWARFParser::CompleteTypeFromDWARF]: Couldn't complete the struct type '!kgen.variant<struct<()>, scalar<bool>, scalar<si64>, scalar<f64>, struct<(pointer<none>, index)>, pointer<none>, pointer<none>, pointer<none>>' because one of its fields has no size. Die = 0x3e9, member die = 0x3f0.
error: JIT(0x7fffa7407000)(0x00007fffa7407000) [MojoDWARFParser::CompleteTypeFromDWARF]: The struct type 'pkg stdlib::pkg builtin::module object::struct object' doesn't have the same size as reported in the DWARF after type completion. Die = 0x3d6.

it also tries to open the stringref.mojo file, but it cannot, because the file is not found

**Comment by hogepodge** [comment]
I'm going to switch this over to be an issue since it looks like a language runtime problem.

**Comment by hogepodge** [comment]
Actually, can I have you file an issue for this? We need some information about version, platform, and how to reproduce it.

**Reply by xeemoff** [reply]
Sure. Describe the data right here?

**Reply by hogepodge** [reply]
It's easiest to have you directly [file an issue here](https://github.com/modularml/mojo/issues/new?assignees=&labels=bug%2Cmojo-repo&projects=&template=mojo_bug_report.yaml&title=%5BBUG%5D). The template includes all of the information you need to provide to the team.
